bin/export-test-case

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+set -euo pipefail
+
+test_case_dir=spec/fixtures/candidate_independent_supporting
+
+sed -n '1p; /trenado/p' downloads/csv/oakland_candidates.csv > $test_case_dir/oakland_candidates.csv
```

Comment:
Maybe you also want to filter for the specific election in case sometime in the future they run again and that changes our test data?

Reply:
Good idea, though this script is meant to be run manually for now just to create the data that gets checked in. Lots of improvement is needed to make this a better UX.